packages/flutter_tools/lib/src/drive/drive_service.dart

```diff
@@ -236,8 +236,13 @@ class FlutterDriverService extends DriverService {
     List<String> browserDimension,
   }) async {
     return _processUtils.stream(<String>[
-      _dartSdkPath,
-      ...<String>[...arguments, testFile, '-rexpanded'],
+      // This is probably not the correct way of calling the executable from
```

Comment:
This won't work reliably because pub will sometimes thing that pub run needs to be repeated